ArgusCore/src/main/java/com/salesforce/dva/argus/service/warden/DummyWardenService.java

```diff
@@ -0,0 +1,196 @@
+import com.salesforce.dva.argus.service.UserService;
+import com.salesforce.dva.argus.service.WardenService;
+import com.salesforce.dva.argus.service.jpa.DefaultJPAService;
+import com.salesforce.dva.argus.system.SystemConfiguration;
+import org.slf4j.Logger;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import javax.persistence.EntityManager;
+
+import static com.salesforce.dva.argus.system.SystemAssert.requireArgument;
+
+/**
+ * Default implementation of the warden service.
+ *
+ * @author  Tom Valine (tvaline@salesforce.com)
+ * @author  Bhinav Sura (bhinav.sura@salesforce.com)
+ */
+
+@Singleton
+public class DummyWardenService extends DefaultJPAService implements WardenService {
+
+    //~ Instance fields ******************************************************************************************************************************
+
+    private ScheduledExecutorService _scheduledExecutorService;
+    @SLF4JTypeListener.InjectLogger
+    private Logger _logger;
+    @Inject
+    private Provider<EntityManager> emf;
+    private final AlertService _alertService;
+    private final MonitorService _monitorService;
+    private final UserService _userService;
+    private final MetricService _metricService;
+    private final ServiceManagementService _serviceManagementRecordService;
+    private final DashboardService _dashboardService;
+    private final AnnotationService _annotationService;
+    private final PrincipalUser _adminUser;
+
+    //~ Constructors *********************************************************************************************************************************
+
+    /**
+     * Creates a new DefaultWardenService object.
+     *
+     * @param  alertService              The alert service user to create warden alerts when necessary. Cannot be null.
+     * @param  monitorService            The monitor service used to collect warden policy metric counters. Cannot be null.
+     * @param  userService               The user service. Cannot be null.
+     * @param  metricService             The metric service. Cannot be null.
+     * @param  serviceManagementService  The service management service. Cannot be null.
+     * @param  dashboardService          The dashboard service. Cannot be null.
+     * @param  auditService              The audit service. Cannot be null.
+     * @param  annotationService         The annotation service. Cannot be null.
+     * @param _sysConfig Service properties
+     */
+    @Inject
+    protected DummyWardenService(AlertService alertService, MonitorService monitorService, UserService userService, MetricService metricService,
+        ServiceManagementService serviceManagementService, DashboardService dashboardService, AuditService auditService,
+        AnnotationService annotationService, SystemConfiguration _sysConfig) {
+        super(auditService, _sysConfig);
+        requireArgument(alertService != null, "Alert service cannot be null.");
```

Comment:
all this body can be removed since all none of this is used in the implementation.